enterprise/server/workflow/service/service.go

```diff
@@ -1281,6 +1286,18 @@ func (ws *workflowService) createApprovalRequiredStatus(ctx context.Context, wf
 	return ghc.CreateStatus(ctx, ownerRepo, wd.SHA, status)
 }
 
+func (ws *workflowService) createQueuedStatus(ctx context.Context, wf *tables.Workflow, wd *interfaces.WebhookData, actionName, invocationID string) error {
+	invocationURL := build_buddy_url.WithPath("/invocation/" + invocationID)
+	invocationURL.RawQuery = "queued=true"
+	status := github.NewGithubStatusPayload(actionName, invocationURL.String(), "Waiting for available worker...", github.PendingState)
```

Comment:
Might this be too long to show up fully in the github UI? Should we just put Queued…?

Reply:
sg, done